src/ui/newmappopup.cpp

```diff
@@ -123,7 +135,11 @@ void NewMapPopup::on_pushButton_NewMap_Accept_clicked() {
     newMap->name = newMapName;
     newMap->type = this->ui->comboBox_NewMap_Type->currentText();
     newMap->location = this->ui->comboBox_NewMap_Location->currentText();
-    newMap->song = "MUS_DAN02";
+    if (projectConfig.getBaseGameVersion() == BaseGameVersion::pokefirered) {
+        newMap->song = "MUS_MASARA";
```

Comment:
these should probably pick from the project's list of {songs, weather, etc}

Reply:
Sure. The first elements of both lists in the projects are `WEATHER_NONE` and `MUS_DUMMY`, which are probably better defaults anyway.